ui/ui_screen.cpp

```diff
@@ -118,6 +118,15 @@ UI::EventReturn UIScreen::OnBack(UI::EventParams &e) {
 	return UI::EVENT_DONE;
 }
 
+#if defined(IOS)
+UI::EventReturn UIScreen::OnBackServer(UI::EventParams &e) {
```

Comment:
Does this really need to be in UIScreen? Can we just implement it in the relevant screen classes in the main PPSSPP project?